docs/technical documentation/standards_and_resources.md

```diff
@@ -0,0 +1,45 @@
+# Tkinter Standards
+
+The GUI layout is closely related to the legacy MATLAB code, but with some improvements for better usability. The majority of the GUI is programmed using Tkinter.
+
+Follow these practices to standardize Tkinter code:
+- Always comment each method.
+- Use object oriented Tkinter.
+- Always use pack or grid immediately after creating a widget.
+- If a widget uses grid layout, perform row and column configuration at the top of the constructor.
+- When a specific padding value is repeated many times, use a constant.
+- Use ttk for most widgets.
+- Use tk for frames.
+- Use tk Scale when discrete slider values are needed.
+- Name files that contain classes that extend frames “frame_”
+- Name files that contain classes that extend windows “window_”
```

Comment:
i think we can get rid of these two, it was used for some parts, but there were other part where it didn't really make sense to use.